tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="SecurityCoordinator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+#pragma warning disable CS0282
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Serilog.Events;
+
+namespace Datadog.Trace.AppSec.Coordinator;
+
+/// <summary>
+/// Bridge class between security components and http transport classes, that calls security and is responsible for reporting
+/// </summary>
+internal readonly partial struct SecurityCoordinator
+{
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<SecurityCoordinator>();
+    private readonly Security _security;
+    private readonly Span _localRootSpan;
+    private readonly HttpTransportBase _httpTransport;
+
+    public bool Blocked => _httpTransport.Blocked;
```

Comment:
tiny non-blocking(!) nit: should use `Is*` prefix here and on `HttpTransportBase` for boolean properties
```suggestion
    public bool IsBlocked => _httpTransport.IsBlocked;
```